crates/agglayer-contracts/src/lib.rs

```diff
@@ -303,19 +303,22 @@ mod tests {
     impl ContractSetup {
         pub fn new() -> Self {
             Self {
-                rollup_manager: "0x32d33D5137a7cFFb54c5Bf8371172bcEc5f310ff" // bali: 0xe2ef6215adc132df6913c8dd16487abf118d1764
+                rollup_manager: "0xE2EF6215aDc132Df6913C8DD16487aBF118d1764" // Bali
                     .parse()
                     .unwrap(),
-                ger_contract: "0xAd1490c248c5d3CbAE399Fd529b79B42984277DF" // bali: 0x2968d6d736178f8fe7393cc33c87f29d9c287e78
+                ger_contract: "0x2968D6d736178f8FE7393CC33C87f29D9C287e78" // Bali
```

Comment:
Was this done for testing or should it stay?

Reply:
This is Bali public GER address, it should stay. The upper one is Cardona.